src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -169,10 +169,7 @@ void FillOutGraphicsPipelineCreateInfo(
         pDstCreateInfo->colorBlendState.blendAttachmentCount = pSrcCreateInfo->outputState.renderTargetCount;
         for (uint32_t i = 0; i < pDstCreateInfo->colorBlendState.blendAttachmentCount; ++i) {
             switch (pSrcCreateInfo->blendModes[i]) {
-                default: {
```

Comment:
can you add an assert in the default case for unknown blend mode?

Reply:
or would it be better to add a new BlendAttachmentState::BlendModeNone() so that all the paths are unified?